src/main/java/seedu/trackermon/ui/ShowDetailsCard.java

```diff
@@ -64,6 +72,8 @@ public void updateShowDetails(Show show) {
             return;
         }
 
+        assert show != null;
+
```

Comment:
Do you think is a good idea to use requireNonNull method here? We used it for many of our methods.

Reply:
I think it is not necessary as my method is able to take in a null show. It is just that I am using a guard clause to break out of the method to prevent it from calling the remaining code (that requires a show to be not null).

Reply:
Alright, no problem